scard.c

```diff
@@ -1671,13 +1671,15 @@ TS_SCardTransmit(STREAM in, STREAM out)
 	cbRecvLength = myCbRecvLength;
 
 	/* FIXME: handle responses with length > 448 bytes */
+#if 0
```

Comment:
If this code is no longer needed, I'd prefer to see it removed. It can be retrieved from the git history if we need it again..

Reply:
Sure. 
I'll fix it ASAP.
Thank you for review.